### PR TITLE
fix: fix tests for strings

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: tests/automation/group_testing.spec.ts [hunk 1]
```diff
@@ -79,21 +81,30 @@ test_group_Alice_1W_Bob_1W_Charlie_1W_Dracula_1W(
...
+      englishStrippedStr('legacyGroupMemberNew')
+        .withArgs({ name: shortenWithBrackets(dracula.accountid) })
+        .toString(),
     );
     await waitForTestIdWithText(
       charlieWindow1,
       'group-update-message',
-      `${dracula.accountid} joined the group.`,
+      englishStrippedStr('legacyGroupMemberNew')
+        .withArgs({ name: shortenWithBrackets(dracula.accountid) })
```
Comment (burtonemily) [inline tests/automation/group_testing.spec.ts:106]: I'll definitely be stealing this function for Appium

**Review by burtonemily — CHANGES_REQUESTED** [review]
Made some comments

File: tests/automation/linked_device_group.spec.ts [hunk 2]
```diff
@@ -59,14 +64,22 @@ test_group_Alice_2W_Bob_1W_Charlie_1W(
...
       'group-update-message',
-      `"${charlie.userName}" has left the group.`,
+      englishStrippedStr('groupMemberLeft')
+        .withArgs({
+          name: charlie.userName,
+        })
+        .toString(),
     );
     // Check for user B
     // Control-message needs to be changed to group-update-message (on disappearing messages branch)
```
[the remark this answers is not in the document]
Comment (Bilb) [reply tests/automation/linked_device_group.spec.ts:74]: yes, removed here and in 2 other places

File: tests/automation/utilities/utils.ts [hunk 3]
```diff
@@ -291,19 +293,9 @@ export async function typeIntoInput(
   const builtSelector = `css=[data-testid=${dataTestId}]`;
   // the new input made with onboarding element needs a click to reveal the input in the DOM
   await clickOnTestIdWithText(window, dataTestId);
-
-  return window.fill(builtSelector, text);
-}
-
-export async function typeIntoInputSlow(
```
[the remark this answers is not in the document]
Comment (Bilb) [reply tests/automation/utilities/utils.ts:298]: wasn't needed. The other one is doing just fine

**Review by burtonemily — APPROVED** [review]

**Pull request merged**